anchore_engine/common/helpers.py

```diff
@@ -28,23 +28,25 @@ def make_response_error(errmsg, in_httpcode=None, details=None):
         ret['detail']['error_codes'] = []
 
     if isinstance(errmsg, Exception):
-        if 'anchore_error_json' in errmsg.__dict__:
-            # Try to load it as json
-            try:
-                err_json = json.loads(errmsg.__dict__.get('anchore_error_json', None))
-            except ValueError:
-                # Then it may just be a string, we cannot do anything with it
-                logger.debug('Failed to parse anchore_error_json as json')
-                return ret
-
-            if {'message', 'httpcode', 'detail'}.issubset(set(err_json)):
-                ret.update(err_json)
+        anchore_error = errmsg.__dict__.get('anchore_error_json', None)
```

Comment:
Non-blocking: I think this should be `getattr(errmsg, 'anchore_error_json', None)` to be more clean than accessing the protected member.